### ISSUE TITLE
Accessibility - My Cities List has Poor Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Closure-App#6]
The My Cities list has poor color contrast.

The lime green check mark (1.44:1) does not have enough color contrast. While the red text and the red plus sign to add cities has a color contrast of 3.7:1 and needs to be 4.5:1 or higher to pass.

See screenshot below.

![my cities list has poor color contrast](https://user-images.githubusercontent.com/10423487/34798942-daa84900-f62b-11e7-8814-6c211a3bac4f.png)

**Comment by georgepliu** [comment]
City names are now black.  This issue has been resolved and is reflected in the current build #11

**Comment by jfc3-dol** [comment]
Thanks, for making the text black, so it has enough color contrast.

I'm going to close this issue.
